src/core/server/rendering/rendering_service.tsx

```diff
@@ -315,16 +315,19 @@ export class RenderingService {
    * @returns {boolean} indicate if the URL is valid/invalid
    */
   public isUrlValid = async (url: string, configName?: string): Promise<boolean> => {
+    if (url === '/') {
```

Comment:
What if the url is empty?

Reply:
In that situation the error should show. We could refactor that its by default empty but by default it is `/` so somebody would have to, in error, set that url to empty.